graylog2-web-interface/src/views/components/aggregationbuilder/AggregationBuilderPropTypes.js

```diff
@@ -21,3 +21,4 @@ export const VisualizationType = PropTypes.string;
 export const VisualizationConfigType = CustomPropTypes.instanceOf(VisualizationConfig);
 
 export const AggregationType = CustomPropTypes.instanceOf(AggregationWidgetConfig);
+export const AggregationData = PropTypes.objectOf(PropTypes.arrayOf(PropTypes.object));
```

Comment:
Maybe we can make it more precise by naming it `AggregationResult`?

Reply:
That is a better name, I've changed it accordingly.